app/views/projects/index/_breadcrumbs.html.erb

```diff
@@ -4,7 +4,7 @@
               class: 'project-folder-link',
               data: { projects_cards_url: cards_projects_url }) %>
   <span>/</span>
-  <% @current_folder.parent_folders.each do |project_folder| %>
+  <% target_folder&.parent_folders&.each do |project_folder| %>
```

Comment:
You can write <% (@current_folder || target_folder).parent_folders.each do |project_folder| %>

Reply:
We have to keep safe navigation. When you are rendering breadcrumbs for the root folder, your target_folder/current_folder is nil

Reply:
Like this?
<% (@current_folder || target_folder)&.parent_folders&.each do |project_folder| %>

Reply:
Yes, but whenever the `@current_folder` exists, `target_folder` is there. I mean, I explicitly passing taget_fodler as a `locale` on all places where I use partial.

Reply:
<% (local_assigns[:target_folder] || @current_folder)&.parent_folders&.each do |project_folder| %>